sql/2023/01/font-display-strategy-usage.sql

```diff
@@ -0,0 +1,76 @@
+# HTTP Archive query to get % of WordPress sites that use various font-display strategy for any web fonts.
+#
+# WPP Research, Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See query results here: https://github.com/GoogleChromeLabs/wpp-research/pull/34
+CREATE TEMPORARY FUNCTION
+  getFontDisplay(json STRING)
+  RETURNS ARRAY < STRING >
+  LANGUAGE js OPTIONS(library = "gs://httparchive/lib/css-utils.js") AS '''
+try {
+  const ast = JSON.parse(json);
+  const result = [];
+  walkDeclarations(ast, decl => {
+    result.push(decl.value);
+  }, {
+    properties: 'font-display',
+    rules: r => r.type === 'font-face'
+  });
+  return result;
+} catch (e) {
+  return [];
+}
+''';
+SELECT
```

Comment:
Let's add a blank line between these two blocks.
```suggestion
''';

SELECT
```